app/views/users/new.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1>Edit</h1>
+<%= render "form", user: @user %>
+
+
```

Comment:
- [ ] You should have two commented out forms: 
  - [ ] The first form uses native `html` tags
  - [ ] The second uses `form_tag` rails helper method for generating html forms,
 => Kindly add them!